include/libc/stddef.h

```diff
@@ -3,18 +3,8 @@
 
 #include "PR/ultratypes.h"
 
-#ifndef NULL
-#define NULL ((void*)0)
-#endif
-
 typedef s32 ptrdiff_t;
 
-#if defined(_MIPS_SZLONG) && (_MIPS_SZLONG == 64)
-typedef unsigned long size_t;
-#else
-typedef unsigned int  size_t;
-#endif
-
```

Comment:
I wouldn't be opposed to have this here too if we wanted to go with the same ifdef checks libultra does